functional/cypress-tests/cypress/fixtures/Widgets.js

```diff
@@ -1,229 +1,225 @@
 module.exports = {
   aemBundleInfo: {
-    name: 'AEM Bundle Info',
```

Comment:
Revert all not related changes.

Reply:
Inside of functional/cypress-tests run `npm install & npx eslint "**/*.js" --fix` and commit all changes. Improvement for this will be done by Tuesday.

Reply:
done